src/components/Match/matchColumns.jsx

```diff
@@ -438,18 +431,18 @@ export const supportColumns = [
 export const chatColumns = [
   {
     displayName: strings.filter_is_radiant,
-    field: 'team',
-    displayFn: row =>
+    displayFn: row => 
```

Comment:
the table will try to read row[field], but if you supply a displayFn then this behavior is overridden.